src/openforms/forms/admin/form_variable.py

```diff
@@ -33,4 +33,5 @@ class FormVariableAdmin(admin.ModelAdmin):
         "data_format",
         "is_sensitive_data",
         "initial_value",
+        "service_fetch_configuration",
```

Comment:
can you add this field to the `raw_id_fields` as well? In general, FK fields should be listed there if you expect > 10 database records, or your admin page will get slower and slower over time to populate the dropdown.